<?xml version="1.0" encoding="UTF-8"?><testsuite failures="5" name="cucumber.runtime.formatter.JUnitFormatter" skipped="0" tests="6" time="110.781206">
<testcase classname="CAPEC-244: Cross-Site Scripting via Encoded URI Schemes" name="Step1.1 Survey the application" time="101.325416">
<failure message="java.lang.AssertionError: &#10;Expected: is &lt;true&gt;&#10;     but: was &lt;false&gt;&#10;&#9;at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)&#10;&#9;at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.outputValide(AppScanningSteps.java:296)&#10;&#9;at ✽.Then output is validated(capec244.feature:14)&#10;"><![CDATA[Given a new scanning session................................................passed
And the application is spidered.............................................passed
Then output is validated....................................................failed
Then input is validated.....................................................skipped

StackTrace:
java.lang.AssertionError: 
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at net.continuumsecurity.steps.AppScanningSteps.outputValide(AppScanningSteps.java:296)
	at ✽.Then output is validated(capec244.feature:14)
]]></failure>
</testcase>
<testcase classname="CAPEC-244: Cross-Site Scripting via Encoded URI Schemes" name="Step2.1 Attempt injection payload variations on input parameters" time="2.763879">
<failure message="net.continuumsecurity.proxy.ProxyException: Error occurred for setScannerAttackStrength&#10;&#9;at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:106)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)&#10;&#9;at ✽.Given prepare to scan cross-site-scripting weakness(capec244.feature:25)&#10;Caused by: org.zaproxy.clientapi.core.ClientApiException: N''existe pas&#10;&#9;at org.zaproxy.clientapi.core.ApiResponseFactory.getResponse(ApiResponseFactory.java:32)&#10;&#9;at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:290)&#10;&#9;at org.zaproxy.clientapi.gen.Ascan.setScannerAttackStrength(Ascan.java:464)&#10;&#9;at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:103)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)&#10;&#9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#10;&#9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#10;&#9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#10;&#9;at java.lang.reflect.Method.invoke(Method.java:498)&#10;&#9;at cucumber.runtime.Utils$1.call(Utils.java:37)&#10;&#9;at cucumber.runtime.Timeout.timeout(Timeout.java:13)&#10;&#9;at cucumber.runtime.Utils.invoke(Utils.java:31)&#10;&#9;at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:37)&#10;&#9;at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)&#10;&#9;at cucumber.runtime.Runtime.runStep(Runtime.java:298)&#10;&#9;at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)&#10;&#9;at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)&#10;&#9;at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:48)&#10;&#9;at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:91)&#10;&#9;at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)&#10;&#9;at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)&#10;&#9;at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#10;&#9;at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#10;&#9;at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#10;&#9;at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#10;&#9;at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#10;&#9;at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#10;&#9;at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)&#10;&#9;at cucumber.api.junit.Cucumber.runChild(Cucumber.java:93)&#10;&#9;at cucumber.api.junit.Cucumber.runChild(Cucumber.java:37)&#10;&#9;at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#10;&#9;at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#10;&#9;at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#10;&#9;at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#10;&#9;at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#10;&#9;at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)&#10;&#9;at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#10;&#9;at cucumber.api.junit.Cucumber.run(Cucumber.java:98)&#10;&#9;at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#10;"><![CDATA[Given prepare to scan cross-site-scripting weakness.........................failed
When the scanner is run.....................................................skipped
And the following false positives are removed...............................skipped
Then no Medium or higher risk vulnerabilities should be present.............skipped
Then output is validated....................................................skipped
Then input is validated.....................................................skipped

StackTrace:
net.continuumsecurity.proxy.ProxyException: Error occurred for setScannerAttackStrength
	at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:106)
	at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)
	at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)
	at ✽.Given prepare to scan cross-site-scripting weakness(capec244.feature:25)
Caused by: org.zaproxy.clientapi.core.ClientApiException: N''existe pas
	at org.zaproxy.clientapi.core.ApiResponseFactory.getResponse(ApiResponseFactory.java:32)
	at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:290)
	at org.zaproxy.clientapi.gen.Ascan.setScannerAttackStrength(Ascan.java:464)
	at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:103)
	at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)
	at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cucumber.runtime.Utils$1.call(Utils.java:37)
	at cucumber.runtime.Timeout.timeout(Timeout.java:13)
	at cucumber.runtime.Utils.invoke(Utils.java:31)
	at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:37)
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)
	at cucumber.runtime.Runtime.runStep(Runtime.java:298)
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:48)
	at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:91)
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:93)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:37)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at cucumber.api.junit.Cucumber.run(Cucumber.java:98)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
]]></failure>
</testcase>
<testcase classname="CAPEC-244: Cross-Site Scripting via Encoded URI Schemes" name="Step3.2 Forceful browsing" time="0.107938">
<system-out><![CDATA[Given a new browser or client instance......................................passed
And vulnerable page with executed script....................................passed
When navigate to inserted link..............................................passed
Then the link is changed....................................................passed
]]></system-out>
</testcase>
<testcase classname="CAPEC-66: SQL Injection" name="Step1.1 Survey application" time="2.448898">
<failure message="java.lang.AssertionError: &#10;Expected: is &lt;true&gt;&#10;     but: was &lt;false&gt;&#10;&#9;at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)&#10;&#9;at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.outputValide(AppScanningSteps.java:296)&#10;&#9;at ✽.Then output is validated(capec66.feature:13)&#10;"><![CDATA[Given a new scanning session................................................passed
And the application is spidered.............................................passed
Then output is validated....................................................failed
Then input is validated.....................................................skipped

StackTrace:
java.lang.AssertionError: 
Expected: is <true>
     but: was <false>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at net.continuumsecurity.steps.AppScanningSteps.outputValide(AppScanningSteps.java:296)
	at ✽.Then output is validated(capec66.feature:13)
]]></failure>
</testcase>
<testcase classname="CAPEC-66: SQL Injection" name="Step2.1 Determine user-controllable input susceptible to injection" time="3.189741">
<failure message="net.continuumsecurity.proxy.ProxyException: Error occurred for setScannerAttackStrength&#10;&#9;at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:106)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)&#10;&#9;at ✽.Given prepare to scan sql-injection weakness(capec66.feature:28)&#10;Caused by: org.zaproxy.clientapi.core.ClientApiException: N''existe pas&#10;&#9;at org.zaproxy.clientapi.core.ApiResponseFactory.getResponse(ApiResponseFactory.java:32)&#10;&#9;at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:290)&#10;&#9;at org.zaproxy.clientapi.gen.Ascan.setScannerAttackStrength(Ascan.java:464)&#10;&#9;at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:103)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)&#10;&#9;at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)&#10;&#9;at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#10;&#9;at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#10;&#9;at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#10;&#9;at java.lang.reflect.Method.invoke(Method.java:498)&#10;&#9;at cucumber.runtime.Utils$1.call(Utils.java:37)&#10;&#9;at cucumber.runtime.Timeout.timeout(Timeout.java:13)&#10;&#9;at cucumber.runtime.Utils.invoke(Utils.java:31)&#10;&#9;at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:37)&#10;&#9;at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)&#10;&#9;at cucumber.runtime.Runtime.runStep(Runtime.java:298)&#10;&#9;at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)&#10;&#9;at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)&#10;&#9;at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:48)&#10;&#9;at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:91)&#10;&#9;at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)&#10;&#9;at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)&#10;&#9;at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#10;&#9;at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#10;&#9;at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#10;&#9;at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#10;&#9;at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#10;&#9;at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#10;&#9;at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)&#10;&#9;at cucumber.api.junit.Cucumber.runChild(Cucumber.java:93)&#10;&#9;at cucumber.api.junit.Cucumber.runChild(Cucumber.java:37)&#10;&#9;at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#10;&#9;at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#10;&#9;at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#10;&#9;at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#10;&#9;at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#10;&#9;at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)&#10;&#9;at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#10;&#9;at cucumber.api.junit.Cucumber.run(Cucumber.java:98)&#10;&#9;at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#10;&#9;at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#10;"><![CDATA[Given prepare to scan sql-injection weakness................................failed
When the scanner is run.....................................................skipped
And the following false positives are removed...............................skipped
Then no Medium or higher risk vulnerabilities should be present.............skipped
Then output is validated....................................................skipped
Then input is validated.....................................................skipped

StackTrace:
net.continuumsecurity.proxy.ProxyException: Error occurred for setScannerAttackStrength
	at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:106)
	at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)
	at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)
	at ✽.Given prepare to scan sql-injection weakness(capec66.feature:28)
Caused by: org.zaproxy.clientapi.core.ClientApiException: N''existe pas
	at org.zaproxy.clientapi.core.ApiResponseFactory.getResponse(ApiResponseFactory.java:32)
	at org.zaproxy.clientapi.core.ClientApi.callApi(ClientApi.java:290)
	at org.zaproxy.clientapi.gen.Ascan.setScannerAttackStrength(Ascan.java:464)
	at net.continuumsecurity.proxy.ZAProxyScanner.setScannerAttackStrength(ZAProxyScanner.java:103)
	at net.continuumsecurity.steps.AppScanningSteps.setAttackStrength(AppScanningSteps.java:305)
	at net.continuumsecurity.steps.AppScanningSteps.enablePolicyy(AppScanningSteps.java:211)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cucumber.runtime.Utils$1.call(Utils.java:37)
	at cucumber.runtime.Timeout.timeout(Timeout.java:13)
	at cucumber.runtime.Utils.invoke(Utils.java:31)
	at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:37)
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)
	at cucumber.runtime.Runtime.runStep(Runtime.java:298)
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:48)
	at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:91)
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:93)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:37)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at cucumber.api.junit.Cucumber.run(Cucumber.java:98)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
]]></failure>
</testcase>
<testcase classname="CAPEC-66: SQL Injection" name="Step2.2 Experiment and try to exploit SQL Injection vulnerability" time="0.945334">
<failure message="java.lang.AssertionError: The user is logged in&#10;Expected: is &lt;false&gt;&#10;     but: was &lt;true&gt;&#10;&#9;at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)&#10;&#9;at net.continuumsecurity.steps.WebApplicationSteps.loginSucceeds(WebApplicationSteps.java:234)&#10;&#9;at ✽.Then the user is logged in(capec66.feature:56)&#10;"><![CDATA[Given a new browser or client instance......................................passed
And the login page..........................................................passed
When force login with 'or'1'='1.............................................passed
Then the user is logged in..................................................failed

StackTrace:
java.lang.AssertionError: The user is logged in
Expected: is <false>
     but: was <true>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at net.continuumsecurity.steps.WebApplicationSteps.loginSucceeds(WebApplicationSteps.java:234)
	at ✽.Then the user is logged in(capec66.feature:56)
]]></failure>
</testcase>
</testsuite>
